src/talk/renderer/DesktopHeader.vue

```diff
@@ -20,74 +20,55 @@
   -->
 
 <template>
-	<header class="header">
-		<!-- Nextcloud Apps use #header selector to get its height -->
-		<!-- This is invisible stub with the same height -->
-		<div id="header" class="header-stub" />
-
-		<a class="header__root-link" @click.prevent="pushToRoot">
```

Comment:
No more option to go back to root with the mouse?

Reply:
Now you can do it with <kbd>Esc</kbd>.

It wasn't obvious anyway, I'm surprised someone else except me even used this feature :D

Reply:
The click is quite common and switching from mouse to keyboard while working one-handed is pretty troublesome.
Can we make the "Nextcloud Talk Preview" navigate to root then when it's displayed anyway?

Reply:
I can, but there is no title on Mac, because it is shown as a separated top bar natively.

Reply:
What's the purpose of navigating to that page @nickvergessen ? 
The sole purpose of that screen is to give some orientation to users when they first open the app, I think it's totally fine if we have no shortcuts to it. 
Are there other plans for that root page that I'm not aware of?

Reply:
> What's the purpose of navigating to that page

Being immediately inactive in all conversations to not appear as online and to receive notifications

Reply:
> Being immediately inactive in all conversations to not appear as online and to receive notifications

I'd say closing the app covers this use case :)